Preface/Preface.pillar

```diff
@@ -58,12 +57,13 @@ available at *http://www.deepintopharo.org*
 
 Do not be frustrated by parts of Pharo that you do not immediately understand.
 You do not have to know everything! Alan Knight expresses this as follows:
-
+{{{latex:\begin{quote} }}}
```

Comment:
@dmitrizagidulin  yes, I'm slowly discovering the background (personally I think it's a kind of NIH syndrom), but I'll do my best of helping us to do good UPBE from typographics point of view even with Pillar. Still remain of opinion for short texts any markup language, for big works LaTeX. "Here I stand. I can do no other"
[two hours long continuation deleted, I know, it's a lost battle, today's people want quick results with minimal effort and so-so quality is good enough]
(I know lawyer, who convert all his docs even to pure TeX for best results :) )